LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2022 Infleqtion
```

Comment:
```suggestion
   Copyright {yyyy} {name of copyright owner}
```
(this is a template that's part of license text; it's not supposed to be filled in)

Reply:
Oh I've been totally doing this wrong elsewhere 😂

Reply:
haha to be fair even qiskit [gets it wrong](https://github.com/Qiskit/qiskit/blob/6a38279c291f49f65c2976039cf209dfe58e9cfe/LICENSE.txt#L191) (as did we up until somewhat recently)

(cirq [gets it right](https://github.com/quantumlib/Cirq/blob/8e4e7d147d6d4ea5dd8111ba21a715aa3acd955c/LICENSE#L189) though)